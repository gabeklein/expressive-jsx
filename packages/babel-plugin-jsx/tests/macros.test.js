transform("output attributes", `
  () => {
    bg: red;
    
    <this />
  };
`);