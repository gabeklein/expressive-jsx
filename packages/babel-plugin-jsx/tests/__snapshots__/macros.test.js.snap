// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`output attributes 1`] = `
  () => <div className="element_7nq" />;
  css(\`
  	.element_7nq {
  		background: red;
  	}
  \`);
`;
